components/search/list-renderer/file.js

```diff
@@ -26,7 +35,20 @@ export default function File({ item: file }) {
         <SearchListItemMeta>
           <div key="lab">{file.lab.title}</div>
           {summary && <div key="summary">{summary}</div>}
+          {file.dbxrefs && <div key="external resources">{file.dbxrefs}</div>}
```

Comment:
You can comma separate these by using `{file.dbxrefs.join(", ")}`. Right now multiple dbxrefs get squished together.

Reply:
Fixed